src/main/webapp/synchronizer.js

```diff
@@ -23,20 +23,21 @@ const TIME_INTERVAL = 100;
  */
 function startVideoSyncTimer() {
   videoSyncTimer = window.setInterval(() => {
-    sync(window.videoPlayer.getCurrentTime());
+    const currentTimeSeconds = window.videoPlayer.getCurrentTime();
+    sync(window.secondsToMilliseconds(currentTimeSeconds));
   }, /* ms= */ TIME_INTERVAL);
 }
 
 /**
- * Calls functions that seek transcript, and discussion to {@code currentTime}
- * (number of seconds since start of video), when the {@code currentTime}
+ * Calls functions that seek transcript, and discussion to `currentTimeMs`
+ * (number of milliseconds since start of video), if the `currentTimeMs`
```

Comment:
If you need to explain what currentTimeMs means then maybe you should use a better name? How about

currentVideoTimeMs?

Reply:
Changed